config/tests/chime_frb_verification.yaml

```diff
@@ -227,58 +233,61 @@ gpu:
     gpu_id: 0
     ew_spacing: [0.0, 0.1, 0.2, 0.3]
     northmost_beam: 90.0
+    commands: *command_list
     in_buffers:
       network_buf: network_buffer_0
       gain_frb_buf: gain_frb_buffer
-      compressed_lost_samples_buf: compressed_lost_samples_buffer_cpy
+      hfb_compressed_lost_samples_buf: compressed_lost_samples_buffer_cpy
     out_buffers:
       beamform_output_buf: gpu_beamform_output_buffer_0
       beamform_hfb_output_buf: gpu_beamform_hfb_output_buffer_0
 
-hyper_fine_beam:
-  num_frames_to_integrate: 5
-  good_samples_threshold: 0.95
-  integrate_hfb_data:
-    kotekan_stage: integrateHFBData
-    log_level: debug
-    hfb_input_buf: gpu_beamform_hfb_output_buffer_0
-    compressed_lost_samples_buf: compressed_lost_samples_buffer
-    hfb_output_buf: beamform_hfb_output_buffer_0
-  
-hfb:
-    kotekan_stage: rawFileWrite
-    in_buf: beamform_hfb_output_buffer_0
-    file_name: 'hfb_buf'
-    file_ext: 'dump'
-    base_dir: '/home/willis/hfb_data'
-
-hfb_read:
-    kotekan_stage: rawFileRead
-    buf: beamform_hfb_output_buffer_read
-    file_name: 'hfb_buf'
-    prefix_hostname: true 
-    file_ext: 'dump'
-    base_dir: '/home/willis/hfb_data'
+#hyper_fine_beam:
+#  num_frames_to_integrate: 5
+#  good_samples_threshold: 0.95
+#  integrate_hfb_data:
+#    kotekan_stage: integrateHFBData
+#    log_level: debug
+#    hfb_input_buf: gpu_beamform_hfb_output_buffer_0
+#    compressed_lost_samples_buf: compressed_lost_samples_buffer
+#    hfb_output_buf: beamform_hfb_output_buffer_0
+#
+#hfb:
+#  kotekan_stage: rawFileWrite
+#  in_buf: beamform_hfb_output_buffer_0
+#  file_name: 'hfb_buf'
+#  file_ext: 'dump'
+#  base_dir: '/home/willis/hfb_data'
+#
+#hfb_read:
+#  kotekan_stage: rawFileRead
+#  buf: beamform_hfb_output_buffer_read
+#  file_name: 'hfb_buf'
+#  prefix_hostname: true
+#  file_ext: 'dump'
+#  base_dir: '/home/willis/hfb_data'
```

Comment:
Why are we commenting out the `HFB` tests?

Reply:
Oh, that was just to get 48k-sample bench, which takes forever to verify. We can set those back to active.